.github/mingw-build.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# set -e
+
+cd "$(dirname "$0")"
+
+buildCyanrip() {
+    cyan_prepare
+    build_curl
+    build_neon
+    build_discid
+    build_libmusicbrainz
+    build_ffmpeg
+    build_cyanrip
+}
+
+cyan_prepare() {
+    local rootdir="$(pwd)"
+    CYANRIPBUILDDIR="$(realpath "$rootdir/_build")"
+    CYANRIPINSTALLDIR="$(realpath "$rootdir/_install")"
+    PKG_CONFIG_PATH=$CYANRIPINSTALLDIR/lib/pkgconfig:$PKG_CONFIG_PATH
+    CPATH="`cygpath -m $CYANRIPINSTALLDIR/include`;`cygpath -m $MINGW_PREFIX/include`"
+    LIBRARY_PATH="`cygpath -m $CYANRIPINSTALLDIR/lib`;`cygpath -m $MINGW_PREFIX/lib`"
+    CPPFLAGS="-D_FORTIFY_SOURCE=0 -D__USE_MINGW_ANSI_STDIO=1"
+    CFLAGS="-mthreads -mtune=generic -O2 -pipe"
+    CXXFLAGS="${CFLAGS}"
+    LDFLAGS="-pipe -static-libgcc -static-libstdc++"
+    export CYANRIPBUILDDIR CYANRIPINSTALLDIR PKG_CONFIG_PATH CPATH LIBRARY_PATH CPPFLAGS CFLAGS CXXFLAGS LDFLAGS
+    mkdir -p "$CYANRIPBUILDDIR" "$CYANRIPINSTALLDIR" || true
+
+    cyan_do_hide_all_sharedlibs
+    sed -ri 's;-R[^ ]*;;g' "$MINGW_PREFIX/lib/pkgconfig/libcdio.pc"
+}
+
+cyan_do_vcs() {
+    local vcsURL=${1#*::} vcsFolder=$2
+    local vcsBranch=${vcsURL#*#} ref=origin/HEAD
+    : "${vcsFolder:=$(basename "$vcsURL" .git)}"
+
+    cd $CYANRIPBUILDDIR
+    git clone "$vcsURL" "$vcsFolder"
+    cd "$vcsFolder"
+}
+
+cyan_create_build_dir () {
+    [[ -d _build ]] && rm -rf ./_build
+    mkdir _build
+    cd _build
+}
+
+cyan_do_hide_all_sharedlibs() {
+    local files
+    files="$(find /mingw{32,64}/lib /mingw{32/i686,64/x86_64}-w64-mingw32/lib -name "*.dll.a" 2> /dev/null)"
+    local tomove=()
+    for file in $files; do
+        [[ -f "${file%*.dll.a}.a" ]] && tomove+=("$file")
+    done
+    printf '%s\n' "${tomove[@]}" | xargs -ri mv -f '{}' '{}.dyn'
```

Comment:
```suggestion
    printf '%s\0' "${tomove[@]}" | xargs -0 -ri mv -f '{}' '{}.dyn'
```

I've heard that using null-terminated can be sometimes be better